src/wwx/2016/Justin Donaldson.md

```diff
@@ -0,0 +1,156 @@
+In general 
+From or for your specific field
+Which creatives/developers/artists (do you admire most? and/or impress you)?
+
+You'll have to hear me rattle on about this during the happy hour :)
+ 
+What contributions are you proud of?
+
+I'm happy that vaxe and promhx still have a following, and that some of my older deprecated libraries have been recomposed as part of larger more general libraries.   This new Lua target is by far my biggest contribution.
+Do you use them in your projects? Which?
+I use vaxe and promhx all the time.
+ 
+Did your contributions bring you work opportunities?
+Yes!  Although sadly I couldn't take advantage of some recent ones, so I won't mention specifics.
+ 
+Tell us about your WWX talk?
+
+I'm going to talk about the new Lua target for Haxe.  If you've been around the Haxe community for a while, you'll know that this is not the first attempt at Lua, and that several efforts fell by the wayside over the years.  I'll talk a bit about why Lua is such a tricky language for Haxe, but also what makes it such a compelling Haxe target.
+
+ 
+What is the best part of WWX for you/are you looking forward too?
+
+There's always one or two presentations that are fantastic, and get me thinking.  I think I started thinking and talking about Lua at the 2014 WWX as I recall.  I like seeing old faces and new faces.  Also, hopefully Franco will arm wrestle someone again.
+ 
+Questions for all previous speakers/attendee's
+Since your last WWX conference as a speaker or attendee, what developments are you impressed by?
+
+The new docs and cookbooks... they're great!
+ 
+What disappoints you, if anything?
+There's still not more US-side Haxe developers, especially in Seattle where I live.
+ 
+Questions for Justin
+At the WWX2014 conference you spoke about promhx, whats changed in the last two years?
+There's been some relatively minor improvements and updates.  I've had several positive but time consuming life changes recently (I have a new 1.5 year old!).  That coupled with the new Lua work has not left me much time to focus on some of my older libraries.
+ 
+While creating the Lua target, whats your experience been like working with the compiler?
+OCaml is strikingly different than just about any language I've used.  I'm still a beginner with it, and my code is not very elegant.  However, I appreciate it more as I use it, and I definitely see precursors to some of my favorite Haxe features there (ADT, helpful typo messages, etc.).  The compiler is comprised of several layers, and I believe the core team has laid out these modules in a sensible fashion.  I don't want to go into too many specifics, but one thing I personally would do differently now would be to rewrite unsupported target expressions where possible, instead of emitting source code directly (as js and lua do currently).  Hopefully that won't bite me to badly with support moving forward.
```

Comment:
I think [this is the commit](https://github.com/HaxeFoundation/haxe/commit/52fa932adae06421378e2d32aaf693acfb97f9d1?diff=unified) that adds cppast.